src/utils/notification.ts

```diff
@@ -35,12 +35,12 @@ export class EmailService {
     };
     try {
       transporter.sendMail(info, (err, inf) => {
-        console.log(err);
+        // console.log(err);
```

Comment:
return this console log back from being uncommented. leave this particular worrying alone.